test/OpenTelemetry.AutoInstrumentation.Loader.Benchmark/OpenTelemetry.AutoInstrumentation.Loader.Benchmark.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="BenchmarkDotNet" VersionOverride="0.13.4" />
```

Comment:
Any possibility to use common version for `BenchmarkDotNet`?
VersionOverride should be avoided if possible.

Reply:
Done.